src/unitTest/InserDataTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+include_once '../dataBase/InsertData.php';
+include_once '../dataBase/XmlFeedModel.php';
+
+use PHPUnit\Framework\TestCase;
+
+class InsertDataTest extends TestCase
+{
+    protected $objectInsert;
+    protected $objectXmlFeed;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->objectInsert = new InsertData();
+        $this->objectXmlFeed = new XmlFeedModel();
+    }
+
+    public function testAddElementToInsert(): void
+    {
+        $this->objectXmlFeed->setTitle(5);
+        $this->objectXmlFeed->setPubDate(10);
+        $this->objectXmlFeed->setLink(8);
+        $this->objectXmlFeed->setDescription(20);
+        $actualNumbers = $this->objectInsert->addElementToInsert($this->objectXmlFeed);
+        $fakeNumbers = [5,10,8,20];
+        $this->assertNotEquals($fakeNumbers, $actualNumbers,'array must have strings not numbers');
+    }
+    public function testPropertyObject(): void
```

Comment:
The name of the test does not give information of what and how exactly being tested.